website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -636,6 +648,18 @@ represents a location outside the Consul cluster. They can be dialed directly wh
                   description: `The number of consecutive failures which cause a host to be
                     removed from the load balancer.`,
                 },
+                {
+                  name: 'EnforcingConsecutive5xx',
+                  type: 'int: 0',
```

Comment:
default values are not quiet right here I think

Reply:
Should the default values be the values mentioned in envoy documentations?